roles/systems/jump_server/tasks/main.yaml

```diff
@@ -81,8 +81,8 @@
 #     - install
 #     - update
 
-- name: Mount /viya-share
-  ansible.builtin.shell: mount -a
+- name: Mount /viya-share # noqa: command-instead-of-module
```

Comment:
There is no ansible module equivalent for mounting all, so command/shell is still recommended here, hence the `noqa`.